extension/src/pages/emby-page.ts

```diff
@@ -0,0 +1,40 @@
+import { VideoDataSubtitleTrack } from '@project/common';
+import { VideoData } from '@project/common';
+
+declare const ApiClient: any | undefined;
+
+document.addEventListener(
+    'asbplayer-get-synced-data',
+    async () => {
+        const deviceID = ApiClient?._deviceId;
+        const apikey = ApiClient?._userAuthInfo.AccessToken;
```

Comment:
If for some reason `ApiClient` doesn't exist then the script should no-op. As it is right now it is possible to `fetch` with `apiKey` being `undefined`.

Reply:
Updated to check `!ApiClient` and send back error in [df1e15f](https://github.com/killergerbah/asbplayer/pull/365/commits/df1e15ffd19897c24ab4a0c383bd707726b83fbc)